core/templates/pages/learner-dashboard-page/new-learner-dashboard-page/home-tab.component.html

```diff
@@ -32,7 +32,11 @@
     <p class="suggested-for-you-heading" [innerHTML]="'I18N_LEARNER_DASHBOARD_SUGGESTED_FOR_YOU_SECTION' | translate">
     </p>
     <div class="empty-suggested-for-you-section" *ngIf="!isNonemptyObject(untrackedTopics)">
-      <p [innerHTML]="'I18N_LEARNER_DASHBOARD_EMPTY_SUGGESTED_FOR_YOU_SECTION' | translate">
+      <p [innerHTML]="'I18N_LEARNER_DASHBOARD_EMPTY_SUGGESTED_FOR_YOU_SECTION_BEFORE_LINK' | translate" class="empty-suggested-for-you-section-text">
+      </p>
+      <p [innerHTML]="'I18N_LEARNER_DASHBOARD_EMPTY_SUGGESTED_FOR_YOU_SECTION_LINK' | translate" class="empty-suggested-for-you-section-text">
```

Comment:
So, this part would be replaced by the <a> link I mentioned in the above comment.

Reply:
Done as per discussion!